### PR TITLE
ocaml-lua.1.6

### PR DESCRIPTION
**Pull request by pdonadeo**
* Tested with OCaml up to 4.07.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
There is a problem on macos:
```
#=== ERROR while compiling ocaml-lua.1.7 ======================================#
# context     2.0.0 | macos/x86_64 | ocaml-base-compiler.4.06.1 | file:///Users/travis/build/ocaml/opam-repository
# path        ~/.opam/4.06.1/.opam-switch/build/ocaml-lua.1.7
# command     ~/.opam/opam-init/hooks/sandbox.sh build ./configure --enable-docs --prefix /Users/travis/.opam/4.06.1 --docdir /Users/travis/.opam/4.06.1/doc/ocaml-lua --libdir /Users/travis/.opam/4.06.1/lib/ocaml-lua
# exit-code   1
# env-file    ~/.opam/log/ocaml-lua-34869-a71bb4.env
# output-file ~/.opam/log/ocaml-lua-34869-a71bb4.out
### output ###
# Exception:
# Sys_error "/tmp/ocaml-lua_build_8d0820.c: Operation not permitted".
# E: Failure("Command './post_conf.ml' terminated with error code 2")
```

File: packages/ocaml-lua/ocaml-lua.1.7/opam [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  [make]
+  [make "doc"]
+]
+remove: [
+    ["ocamlfind" "remove" "lua"]
+    ["rm" "-R" "%{doc}%/ocaml-lua"]
+]
+depends: [
+  "ocaml" {>= "4.02.1" & < "4.08.0"}
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/ocaml-lua/ocaml-lua.1.7/opam]: I can't see any use of the `Findlib` library in your code. Could you add the `{build}` attribute if you don't use it?

Comment (pdonadeo) [reply packages/ocaml-lua/ocaml-lua.1.7/opam]: findlib is actually used to install and remove the library, I'm going to add the `{build}` constraint.

Comment (pdonadeo) [reply packages/ocaml-lua/ocaml-lua.1.7/opam]: About OSX: is the output file `~/.opam/log/ocaml-lua-34869-a71bb4.out` available for a check? It seems the configure script can't write in `/tmp` but I'm not sure at all.

**Comment by pdonadeo** [comment]
@kit-ty-kate The library compiles at least on one OSX: https://discuss.ocaml.org/t/compiling-ocaml-lua-under-osx/2789/5

**Comment by mseri** [comment]
I have tested it on two more machines and seems to work. Merging

**Comment by pdonadeo** [comment]
@mseri thanks :smiley:

**Pull request merged**